test/src/unit-capi-consolidation.cc

```diff
@@ -6355,13 +6213,6 @@ TEST_CASE_METHOD(
     ConsolidationFx,
     "C API: Test consolidating fragment metadata, sparse string",
     "[capi][consolidation][fragment-meta][sparse][string]") {
```

Comment:
Did we forget the REST tag here?

Reply:
yes, forgot to add this test since it applies to REST, good catch!

Reply:
This turned out to need significant work as I needed to adapt the whole consolidation test suite so that I can run those few tests (`fragment_metadata` and `commits` consolidation) on REST CI. I did it because I think it's worth it as we can now very easily enable the rest of the tests to run on REST CI when we enable fragment consolidation on for remote arrays.